docs/installation.md

```diff
@@ -58,11 +58,22 @@ The following instructions are for setting up a version of Amundsen using Docker
     $ docker-compose -f docker-amundsen.yml down
     ```
 
-## Verify setup or troubleshooting
+## Verify setup
 
 1. You can verify dummy data has been ingested into Neo4j by by visiting [`http://localhost:7474/browser/`](http://localhost:7474/browser/) and run `MATCH (n:Table) RETURN n LIMIT 25` in the query box. You should see two tables:
    1. `hive.test_schema.test_table1`
    2. `dynamo.test_schema.test_table2`
 2. You can verify the data has been loaded into the metadataservice by visiting:
    1. [`http://localhost:5000/table_detail/gold/hive/test_schema/test_table1`](http://localhost:5000/table_detail/gold/hive/test_schema/test_table1)
-   2. [`http://localhost:5000/table_detail/gold/dynamo/test_schema/test_table2`](http://localhost:5000/table_detail/gold/dynamo/test_schema/test_table2)
\ No newline at end of file
+   2. [`http://localhost:5000/table_detail/gold/dynamo/test_schema/test_table2`](http://localhost:5000/table_detail/gold/dynamo/test_schema/test_table2)
+
+## Troubleshooting
+
+1. If the docker VM doesn't have enough heap memory for Elastic Search, `es_amundsen` will quietly fail during `docker-compose` and cause errors in the `sample_data_loader.py` script.
```

Comment:
Nit: replace the term “VM” with “container”.

Suggestion: If we remove the `-d` parameter in the Docker compose up step under installation the failing will be less silent. Down the road it may also make other errors in logs become more visible which is good. Finally removing `-d` also allows us to get rid of the `docker-compose down` step, as you just Ctrl-C.

Happy to open a separate issue for the last suggestion if you think it warrants to be discussed on it’s own.

Reply:
Discussed with team offline, removed `-d` option.